yagb-core/src/emulator/ppu/ppu-cgb.ts

```diff
@@ -5,6 +5,7 @@ import { PpuBase } from './ppu-base';
 import { Savestate } from '../savestate';
 import { SpriteQueueDmg } from './sprite-queue-dmg';
 import { cgbRegisters } from '../cgb-registers';
+import {ppuMode} from "../ppu";
```

Comment:
Seems prettier is not running properly on your end 😏 Maybe give it a spin manually.